internal/actions/release.go

```diff
@@ -28,7 +28,7 @@ func Release() error {
 
 	dir := "."
 	usingReleasesMd := false
-	// TODO: Maybe instead check for workflow dispatch event
+	// We do not currently support specific target workflow dispatch for terraform. This is because terraform relies on release history.
```

Comment:
This comment doesn't seem to align with what's going on below? Where is terraform mentioned?

Reply:
The workflow_dispatch won't be setup in speakeasy configure publishing for terraform. I can also add an error state here though.

Reply:
Updated. I think it makes more sense now